### PR TITLE
Created upload file in admin and settings module with upload function

### LINKED ISSUE #301: Remove uploads from app.py
Find a way to include codes in module. Maybe add upload.py with function for box and module

### PR DESCRIPTION
**Pull request by Yahyakiani**
Created Upload file in setting and admin module with separate upload function for each.
Resolves #301 
test

### CODE REVIEW

**Comment by rehmanis** [comment]
@Yahyakiani I checked and it does not compile. Run these commands in order:
```bash
python manage.py clean
python manage.py initialise
```

To fix, you will have to move the following code (in cmd.py) to uploads function of respective module(setting and admin) and remove the arguments from upload functions in these module i.e. `upload()` instead of `upload(email, password)` for example.

```python

def initialise():

   #..... some code here not shown

    with open("config.json", "r") as config:
        config = json.load(config)

   #..... some code here not shown

    print("Initialising User")
    print(SEP_CHAR * SEP_NUM, end="\n\n")
    add_admin(config["admin_user"]["email"], config["admin_user"]["password"])

    print("Initialising Settings")
    print(SEP_CHAR * SEP_NUM, end="\n\n")
    for name, value in config["settings"].items():
        add_setting(name, value)
```

In the above, the with `    with open("config.json", "r") as config....:` code block needs to be added to both the upload.py file of the module(settings, and admin).

finally remove the uploads.py file in the shopyoapi. Also remove all `flake8` (pep8speaks message above)errors. See step 8 of https://www.appinv.science/shopyo/contrib.html#making-a-pull-request

**Comment by Yahyakiani** [comment]
@rehmanis Should i remove the conflicts by merging the latest dev changes in this branch or will you resolve them while merging ?
Because  I am unable to find the following two alerts in my files in the current status.

**Comment by rehmanis** [comment]
@Yahyakiani yes do the following as mentioned here:

```
git checkout dev
git fetch upstream
git pull upstream dev
git checkout remove_uploads
git --no-ff merge dev
// resolve conflicts, add and then commit the new changes. Finally push these new changes
git push origin remove_uploads
```

Also remove the with clause in the cmd.py:
```python
def initialise():

   #..... some code here not shown
    
    # remove this as we no longer using it and it causes alert
    with open("config.json", "r") as config:
        config = json.load(config)

   #..... some code here not shown
```
Let me know what conflicts you see. When I do merge, I was not seeing any conflicts. If you are unable to do so or having lots of issues with merging then I can do it.

File: shopyo/shopyoapi/cmd.py [hunk 1]
```diff
@@ -91,14 +91,14 @@ def initialise():
         [sys.executable, "manage.py", "collectstatic"], stdout=subprocess.PIPE
     )
 
-    print("Initialising User")
-    print(SEP_CHAR * SEP_NUM, end="\n\n")
-    add_admin(config["admin_user"]["email"], config["admin_user"]["password"])
+    # print("Initialising User")
+    # print(SEP_CHAR * SEP_NUM, end="\n\n")
+    # add_admin(config["admin_user"]["email"], config["admin_user"]["password"])
```
Comment (rehmanis) [inline shopyo/shopyoapi/cmd.py]: Delete this commented out code as well as the other one for settings.

**Comment by rehmanis** [comment]
@Yahyakiani perfect. All good. One last thing, if you can remove the two unused imports as shown in https://lgtm.com/projects/g/Abdur-rahmaanJ/shopyo/rev/pr-479368c35f0f4595faab3932f075b84a7c34ad25 that would be great. I will merge it once that is done.

**Pull request merged**